api/src/main/java/org/openmrs/module/appointments/service/impl/TeleconsultationAppointmentSavedEventListener.java

```diff
@@ -32,4 +31,12 @@ public void onApplicationEvent(TeleconsultationAppointmentSavedEvent event) {
             log.error("Unable to send teleconsultation appointment email notification", e);
         }
     }
+
+    public TeleconsultationAppointmentNotificationService getEmailNotificationService() {
+        return emailNotificationService;
+    }
+
+    public void setEmailNotificationService(TeleconsultationAppointmentNotificationService emailNotificationService) {
+        this.emailNotificationService = emailNotificationService;
+    }
```

Comment:
@angshu not sure if we need to confirm but this is required by Spring I "think".